### PR TITLE
Support library:// and shub:// from singularity run/exec/shell

### LINKED ISSUE #2098: Run from Library Not Working
With current `master` (`v3.0.0-beta.1-8-gccc859dd`), I am not able to run directly from a Library image:

```sh
$ singularity run library://alpine
FATAL:   Failed to get SHA of library://alpine: Unable to parse image name library://alpine: library not a registered transport
```

### PR DESCRIPTION
**Pull request by bauerm97**
**Description of the Pull Request (PR):**

Title

**This fixes or addresses the following GitHub issues:**

- Fixes #2098

### CODE REVIEW

File: src/cmd/singularity/cli/actions.go [hunk 1]
```diff
@@ -87,37 +90,85 @@ func init() {
...
+		return
+	}
+
+	var image string
+	var err error
+
+	switch t {
+	case uri.Library:
+		sylabsToken(cmd, args) // Fetch Auth Token for library access
+
```
Comment (ArangoGutierrez) [inline src/cmd/singularity/cli/actions.go:165]: white line

**Review by ArangoGutierrez — CHANGES_REQUESTED** [review]
LGTM , just a minor thing

**Comment by ArangoGutierrez** [comment]
```bash
[eduardo@rhel7 tmp]$ singularity run library://sylabsed/examples/lolcow:latest
INFO:    Downloading library image
 89.13 MiB / 89.13 MiB [===================================================================================================================================================================] 100.00% 6.15 MiB/s 14s
 ____________________________________
/ He is now rising from affluence to \
| poverty.                           |
|                                    |
\ -- Mark Twain                      /
 ------------------------------------
        \   ^__^
         \  (oo)\_______
            (__)\       )\/\
                ||----w |
                ||     ||
```
works for me 
RHEL7

File: src/cmd/singularity/cli/actions.go (hunk 1 above)
Comment (bauerm97) [reply src/cmd/singularity/cli/actions.go:165]: I put that there intentionally

Comment (ArangoGutierrez) [reply src/cmd/singularity/cli/actions.go:165]: got it

**Review by ArangoGutierrez — APPROVED** [review]

**Pull request merged**